src/main.vala

```diff
@@ -15,7 +15,7 @@ int main (string[] args) {
     }
 
     if (argMap.has_key ("-v") || argMap.has_key ("--version")) {
-        print ("remontoire 1.3.0 (C) 2020 Ken Gilmer\n");
+        print ("remontoire 1.4.0 (C) 2020 Ken Gilmer\n");
```

Comment:
Thanks @gardotd426 !  Yeah the actual version is specified in the [Debian file `changelog`](https://github.com/regolith-linux/remontoire/blob/master/debian/changelog#L1) and the current version is actually `1.4.1`.  Can you update that?  (Ideally we would have something in the build that would read the version from the changelog file and replace the version in the source but :shrug:

Reply:
Absolutely I'll change it as soon as I finish lunch.